frontend/src/components/authentication/AuthenticationTabs.jsx

```diff
@@ -1,26 +1,13 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import TabNavigation from "../commons/TabNavigation";
 import LoginForm from "./LoginForm";
 import CreateAccountForm from "./CreateAccountForm";
 import LOCALIZE from "../../text_resources";
+import { Tabs, Tab, Container, Row, Col } from "react-bootstrap";
 
 const styles = {
-  container: {
-    width: 500,
-    margin: "0px auto",
-    paddingTop: 24,
-    paddingRight: 20,
-    paddingLeft: 20
-  },
-  tabNavigationStyle: {
-    height: "100%",
-    backgroundColor: "white",
-    borderWidth: "1px 1px 1px 1px",
-    borderStyle: "solid",
-    borderColor: "#00565e",
-    borderTopColor: "white",
```

Comment:
Do we really want no border here?

![image](https://user-images.githubusercontent.com/23021242/57222253-e3aa1a00-6fcf-11e9-877c-878496597f27.png)

I find that a little strange to have no border in a form with a white background on a light grey background page.

Reply:
Oops - thanks for finding! Fixed in latest commit.